### ISSUE TITLE
Question on validation loss

### ISSUE THREAD

**Issue by Sirius083** [YixuanLi/densenet-tensorflow#20]
I have implemented your code on cifar100
when halving the learning rate, the test loss suddenly decrease , then increase
while the validation error keeps unchanged.
Did tensorpack maintain a shadow value and show the EMA of cost? (reset this value after learning rate change)
![cifar100_d_40_k_12](https://user-images.githubusercontent.com/22949166/57216349-2f85af00-7022-11e9-8449-60e69fe1f399.png)

Any suggesstion would be appreciated
Thanks in advance
